.github/workflows/check-go-version.yml

```
@@ -0,0 +1,72 @@
+        
+        # バージョン番号のみを抽出（go1.24.4 -> 1.24）
+        LATEST_MINOR=$(echo $LATEST_VERSION | sed 's/^go//' | cut -d. -f1,2)
+        echo "latest_minor=$LATEST_MINOR" >> $GITHUB_OUTPUT
+        echo "Latest minor version: $LATEST_MINOR"
+
+    - name: Check if GO_LATEST is up to date
+      run: |
+        # ci.ymlからGO_LATESTの値を取得
+        CURRENT_VERSION=$(grep "GO_LATEST:" .github/workflows/ci.yml | awk '{print $2}' | sed 's/\.x$//')
+        echo "Current GO_LATEST: $CURRENT_VERSION"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)